tests/ftests/051-sudo-cgroup_get_cgroup.py

```diff
@@ -14,7 +14,7 @@
 import sys
 import os
 
-CGNAME = '051cgnewcg'
```

Comment:
We need to use a child cgroup here because we don't know what controllers will be enabled in the root's `cgroup.subtree_control` file and this could vary from one system to another.  Using a child cgroup lets us explicitly dictate the enabled controllers.